build-system/pr-check.js

```diff
@@ -272,6 +272,7 @@ function stopSauceConnect() {
 const command = {
   testBuildSystem: function() {
     timedExecOrDie('gulp ava');
+    timedExecOrDie('node node_modules/jest/bin/jest.js');
```

Comment:
The output of this task is fairly verbose: https://travis-ci.org/ampproject/amphtml/jobs/408608717#L647-L664. Any chance it can be silenced on Travis in the passing case? (similar to all our other checks)

Also, does it make sense to add a `gulp` task for this, so that it can be run along with other checks?

Reply:
honestly, im trying to avoid gulp LOL. this code will hopefully be separated out some point into its own module.

will change the output

Reply:
done. on the simple output. using a dot reporter

Reply:
I was initially suggesting the use of https://www.npmjs.com/package/gulp-jest to maintain consistency across our toolchain. But I realize it's not ideal to continue using `gulp`.

Less verbose output LGTM, although I don't see any dots: https://travis-ci.org/ampproject/amphtml/jobs/408625988#L648-L653

Reply:
yeah looks like it only outputs dots for test suites and since theres only 1 suite (with 8 tests) it doesnt output any dots.

Reply:
No worries! `</bikeshed>` :)